bokeh/sphinxext/bokeh_plot.py

```diff
@@ -170,8 +171,16 @@ def run(self):
         target = nodes.target("", "", ids=[target_id])
         result = [target]
 
+        # if this is NOT an inline example and a docstring exists, add it
+        if not self.content and doc:
```

Comment:
Second guessing myself, but I guess if it actually works with inline content, maybe there is no reason to go out of our way to prevent it. So then this could just be `if doc:` (and then I think the comment can also be removed)